packages/ddp-client/src/livechat/LivechatClientImpl.ts

```diff
@@ -359,6 +359,11 @@ export class LivechatClientImpl extends DDPSDK implements LivechatStream, Livech
 		return this.rest.put(`/v1/livechat/message/${id}`, params);
 	}
 
+	unsubscribeAll(): Promise<unknown> {
+		const subscriptions = Array.from(this.client.subscriptions.keys());
+		return Promise.all(subscriptions.map((subscription) => this.client.unsubscribe(subscription)));
```

Comment:
Can we just 
```suggestion
		return Promise.all(subscriptions.map(this.client.unsubscribe));
```
or would it mess up with the types?

Reply:
@KevLehman It wouldn't, I even considered it, but I'm not sure if `unsubscribe` uses `this` in its internal implementation. Didn't think the risk was worth 😬